File: Backend/models/Report.js
const mongoose = require('mongoose');

const ReportsSchema = new mongoose.Schema({
    id : {type:String},
    idUsuario : {type: String, require: true},
    categoria : {type: String, default: 'Otros'},
    foto : {type:String, default: "null"},
    fechaReporte:{type: Date, require: true },
    ubicacion : {type:String, require: true},
    descripcion : {type:String, require: true},
    estado : {type:String, require: true}
});

// 'reports' is the name of the collection
module.exports = new mongoose.model('reports', ReportsSchema);

File: Frontend/app/src/main/java/com/example/tamayoreport/model/entities/Report.kt
package com.example.tamayoreport.model.entities

import com.google.gson.annotations.SerializedName
import java.util.*

class Report (
    @SerializedName("_id")
    var id: String,
    var categoria: String,
    var foto: String?,
    var fechaReporte: String,
    var ubicacion: String,
    var descripcion: String,
    var estado:String
){
}